server/api/reviews.js

```diff
@@ -19,4 +19,11 @@ router.get('/:productId', (req, res, next) => {
     .catch(next);
 });
 
+// api/reviews: create review
+router.post('/', (req, res, next) => {
+  Review.create(req.body)
+    .then(review => res.status(201).send(review))
```

Comment:
Don't need to send status, but does not hurt if you do.

```suggestion
    .then(review => res.send(review))
```